tornettools/generate.py

```diff
@@ -133,14 +141,57 @@ def __get_scaled_tgen_server_bandwidth_kbit(args):
     scaled_bw = scaled_client_bw * n_clients_per_server
     return scaled_bw
 
-def __server(args, server):
+def __filter_nodes(network, ip_address_hint, country_code_hint):
+    all_nodes = [{'id': node[0], **node[1]} for node in network.nodes(data=True)]
```

Comment:
Interesting - I've never seen the `**node[1]` syntax before. Maybe add a comment?

Reply:
Added a comment here.